src/main/java/codechicken/nei/ItemPanel.java

```diff
@@ -166,19 +166,32 @@ public int getHeight(GuiContainer gui) {
     }
 
     protected int resizeFooter(GuiContainer gui) {
-        if (!NEIClientConfig.showItemQuantityWidget()) return 0;
+        if (!(NEIClientConfig.showItemQuantityWidget() || !NEIClientConfig.isSearchWidgetCentered())) return 0;
 
         final int BUTTON_SIZE = 16;
 
         more.w = less.w = BUTTON_SIZE;
-        more.h = less.h = quantity.h = BUTTON_SIZE;
 
-        less.x = x;
-        more.x = x + w - BUTTON_SIZE;
-        more.y = less.y = quantity.y = y + h - BUTTON_SIZE;
-
-        quantity.x = x + BUTTON_SIZE + 2;
-        quantity.w = more.x - quantity.x - 2;
+        if (NEIClientConfig.isSearchWidgetCentered()) {
+            more.x = x + w - BUTTON_SIZE;
+            more.h = less.h = quantity.h = BUTTON_SIZE;
+            more.y = less.y = quantity.y = y + h - BUTTON_SIZE;
+            less.x = x;
+            quantity.x = x + BUTTON_SIZE + 2;
+            quantity.w = more.x - quantity.x - 2;
+        } else {
+            more.x = less.x;
+            more.y = y + h - BUTTON_SIZE;
+            more.h = less.h = BUTTON_SIZE / 2;
+
+            less.x = (int) (x + (w * 0.7));
```

Comment:
Should this coefficient be constant (0.7) or depend on `NEIClientConfig.showItemQuantityWidget()`?